docs/python/design.md

```diff
@@ -27,11 +27,11 @@ The Azure SDK should be designed to enhance the productivity of developers conne
 * The Azure SDK for each target language feels like a single product developed by a single team.
 * There should be feature parity across target languages. This is more important than feature parity with the service.
 
-#### Approachable
+#### 접근하기쉽게
 
-* We are experts in the supported technologies so our customers, the developers, don't have to be.
-* Developers should find great documentation (hero tutorial, how to articles, samples, and API documentation) that makes it easy to be successful with the Azure service.
-* Getting off the ground should be easy through the use of predictable defaults that implement best practices. Think about progressive concept disclosure.
+* 우리는 지원되는 기술들의 전문가로 우리의 고객들과 개발자들은 전문성을 가질 필요가 없습니다.
+* 개발자들은 Azure 서비스를 성공적으로 사용하기 쉽게 해주는 좋은 문서(튜토리얼, 방법 문서, 샘플들 및 API 문서)를 찾아야 합니다.
+* 모범 사례를 구현하는 예측 가능한 기본값들을 사용함으로써 쉽게 시작할 수 있습니다. Think about progressive concept disclosure.
```

Comment:
Not sure why English and Korean are mixed into one line

Reply:
The modification has been completed. Thank you for your feedback.